### DISCUSSION TITLE
[求助]今天的驱动盘本是不是有问题？

### DISCUSSION THREAD

**Discussion by tmtzsj** [share121/inter-knot#1773] [其他]
谁能帮我找找我的极品冰套掉到哪里了？
![微信图片_20240812214910](https://github.com/user-attachments/assets/dc135787-cb8d-49ae-9846-53350e6c2430)
![微信图片_20240812214916](https://github.com/user-attachments/assets/80a97aa1-6501-4ae3-b1a8-ee0b0c0140b7)
![微信图片_20240812214922](https://github.com/user-attachments/assets/019dd1fa-f042-4837-aa17-f0a3d5d9ea72)

**Comment by 42417** [comment]
把右上角带有“new”的一个个排查不就好了（doge
